homeassistant/components/sonos/switch.py

```diff
@@ -321,12 +320,12 @@ def available(self) -> bool:
         return (self.alarm is not None) and self.speaker.available
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return state of Sonos alarm switch."""
         return self.alarm.enabled
 
     @property
-    def extra_state_attributes(self):
+    def extra_state_attributes(self) -> dict[str, str | bool | float]:
```

Comment:
```suggestion
    def extra_state_attributes(self) -> dict[str, Any]:
```
Usually simpler to type the value portion as `Any` instead.